spec/system/solr_tasks_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe "Solr Indexing Tasks", type: :system do
```

Comment:
This test is looking great! 

Would you add "clean: true" info to this test? This makes it so that the Solr index is cleaned out before each "it" statement, so that we can be sure we don't get into an order-dependent test state with our Solr index. See the FixtureIndexingService spec for an example.